easybuild/easyconfigs/c/CMake/CMake-3.4.1-foss-2015b.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.4.1'
+
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('ncurses', '6.0')]
```

Comment:
because ncurses is a very common dependency, sticking to 5.9 for now is probably better; if not, you won't be able to load this CMake module together with lots of other stuff that depends on ncurses 5.9

we plan to consistently switch to ncurses 6.0 with the 2016a toolchains